fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dto/ReferenceParmVal.java

```diff
@@ -21,11 +22,14 @@
     private String name;
 
     // The reference value
-    private String valueString;
+    //private String valueString;
```

Comment:
clean up commented out lines (here and 48-54)

Reply:
thanks...cruft left over from some refactor back-and-forth

Reply:
done